src/panel/graph/AudioGraphRender.ts

```diff
@@ -73,6 +73,8 @@ export class AudioGraphRender {
 
     this.graphicsCache = new GraphicsCache();
 
+    this.graphicsCache = new GraphicsCache();
```

Comment:
Looks like a duplicate of L74.

Reply:
Yes. That's a duplicate.

Merging main into this PR made some conflicts. This ... didn't appear as one of those oddly.